package.json

```diff
@@ -55,10 +51,11 @@
   },
   "scripts": {
     "build": "tsc",
-    "check": "npm-check",
     "clean": "rimraf coverage dist",
     "clean:build": "rimraf dist",
     "clean:test": "rimraf coverage",
+    "format": "prettier --write .",
+    "lint": "echo Currently no linter installed",
```

Comment:
I'm hoping these will be changed to `@linter/cli` in the future :D

Reply:
Exactly 🥰